mne/tests/test_epochs.py

```diff
@@ -346,6 +346,7 @@ def test_hierarchical():
 
 
 @pytest.mark.slowtest
+@pytest.mark.filterwarnings("ignore:All epochs were dropped!.*:RuntimeWarning")
```

Comment:
Rather than filtering for the entire test, it would be better to target the hopefully one or two lines that actually produce no epochs. If it's more than that number of calls the test is probably buggy and I can take a look.